certified-connectors/IN-D Payables/apiDefinition.swagger.json

```diff
@@ -0,0 +1,365 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "IN-D Payables",
+    "description": "IN-D Payables process invoices to extract only the useful details like invoice numbers, PO number, vendor name, etc., and the line items in the table automatically without a need to input a template. The IN-D Payables, can process multiple invoices at once directly from the source you choose from, and send the relevant fields to the accounting software connected in the flow for accounting purposes in a matter of seconds.",
+    "version": "1.0",
+    "contact": {
+      "name": "IN-D Support",
+      "url": "https://one.in-d.ai/developer/",
+      "email": "explore@in-d.ai"
+    }
+  },
+  "host": "api.payables.in-d.ai",
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/extract_data": {
+      "post": {
+        "responses": {

```

Comment:
@gowthamd3 The swagger file is not the updated one - Can you please upload the one you have uploaded in the certification portal